config/main.py

```diff
@@ -395,14 +435,32 @@ def _get_platform():
                 return tokens[1].strip()
     return ''
 
+def _get_num_asic():
+    platform = _get_platform()
+    asic_conf_file = os.path.join('/usr/share/sonic/device/', platform, ASIC_CONF_FILENAME)
+    if not os.path.isfile(asic_conf_file):
+        num_asic=1
```

Comment:
the indent is not consistent, should have space around the '='

Reply:
Fixed